File: challenges/2-idade-em-dias/python/marlonmartins2/idade.py
def calcula_idade(dias):
    anos = dias // 365
    meses = (dias % 365) // 30
    dias = (dias % 365) % 30
    return f"""
        {anos} ano(s)
        {meses} mes(es)
        {dias} dia(s)
    """

if __name__ == "__main__":
    dias = int(input("Digite a quantidade de dias: "))
    print(calcula_idade(dias))

File: challenges/2/elixir/sherlockzen/idade.exs
defmodule Idade do
  # def iniciar() do
  #   IO.gets("Digite o valor inteiro correspondente aos dias da idade desejada: ")
  #   |>Integer.parse()|>Idade.verificavalor()|>Idade.idadedias()
  # end

  # def verificavalor(:error) do
  #     IO.puts("Valor inválido!")
  #     Idade.iniciar()
  # end
  # def verificavalor({num, _}) do
  #     num
  # end

  def idadedias(valor) do
    anos = div(valor, 365)
    meses = div(rem(valor, 365), 30)
    dias = rem(rem(valor, 365), 30)

    IO.puts("#{anos} ano(s)\n#{meses} mes(es)\n#{dias} dia(s)")
  end

end
